site/tutorials/gettings_started.md

```diff
@@ -0,0 +1,293 @@
+---
+layout: tutorials
+menu: tutorials
+title: Introduction to Vega-Lite
+permalink: /tutorials/getting_started.html
+---
+
+In this tutorial, you will learn how to write a visualization specification in Vega-Lite. You will start with a simple visualization, which you will then extend the encoding and define additional properties. Lastly, you will learn how to embed the final visualization on a web page. There are similar tutorials for [D3](http://bost.ocks.org/mike/bar/) and [Vega](https://github.com/vega/vega/wiki/Tutorial).
```

Comment:
I like to have them because people know the d3 tutorial and may have seen the veg tutorial. Provides context.